<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GroomTool</class>
 <widget class="QWidget" name="GroomTool">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>245</width>
    <height>449</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
/*
QWidget{
background: rgb(255, 255, 127)
}
*/
</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="2" column="0">
      <widget class="QCheckBox" name="fill_holes_checkbox">
       <property name="text">
        <string>Fill Holes</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QCheckBox" name="center_checkbox">
       <property name="text">
        <string>Center</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QCheckBox" name="isolate_checkbox">
       <property name="text">
        <string>Isolate</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QCheckBox" name="autocrop_checkbox">
       <property name="text">
        <string>Autocrop</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QCheckBox" name="autopad_checkbox">
       <property name="text">
        <string>Pad</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QGroupBox" name="pad_groupbox">
       <property name="title">
        <string/>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Amount</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="padding_amount">
          <property name="maximum">
           <number>500</number>
          </property>
          <property name="value">
           <number>10</number>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QCheckBox" name="antialias_checkbox">
       <property name="text">
        <string>Antialias</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QGroupBox" name="antialias_groupbox">
       <property name="title">
        <string/>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Iterations</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="antialias_iterations">
          <property name="maximum">
           <number>500</number>
          </property>
          <property name="value">
           <number>10</number>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QCheckBox" name="fastmarching_checkbox">
       <property name="text">
        <string>Fastmarching</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="9" column="0">
      <widget class="QGroupBox" name="fastmarching_groupbox">
       <property name="title">
        <string/>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <property name="topMargin">
           <number>0</number>
          </property>
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Iso Value</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="iso_value">
            <property name="minimum">
             <double>-999999999.000000000000000</double>
            </property>
            <property name="maximum">
             <double>999999999.000000000000000</double>
            </property>
            <property name="value">
             <double>0.500000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <property name="topMargin">
           <number>0</number>
          </property>
          <item>
           <spacer name="horizontalSpacer_5">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="sigma2">
            <property name="text">
             <string>Sigma</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="fastmarch_sigma"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item row="10" column="0">
      <widget class="QCheckBox" name="blur_checkbox">
       <property name="text">
        <string>Blur</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="11" column="0">
      <widget class="QGroupBox" name="blur_groupbox">
       <property name="title">
        <string/>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Sigma</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="blur_sigma">
          <property name="value">
           <double>2.000000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="export_xml_button">
       <property name="text">
        <string>Export XML</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="skipButton">
       <property name="text">
        <string>Skip</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="run_groom_button">
       <property name="text">
        <string>Run Groom</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>center_checkbox</tabstop>
  <tabstop>isolate_checkbox</tabstop>
  <tabstop>fill_holes_checkbox</tabstop>
  <tabstop>autocrop_checkbox</tabstop>
  <tabstop>autopad_checkbox</tabstop>
  <tabstop>padding_amount</tabstop>
  <tabstop>antialias_checkbox</tabstop>
  <tabstop>antialias_iterations</tabstop>
  <tabstop>fastmarching_checkbox</tabstop>
  <tabstop>iso_value</tabstop>
  <tabstop>fastmarch_sigma</tabstop>
  <tabstop>blur_checkbox</tabstop>
  <tabstop>blur_sigma</tabstop>
  <tabstop>export_xml_button</tabstop>
  <tabstop>skipButton</tabstop>
  <tabstop>run_groom_button</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
